pyspeckit/cubes/SpectralCube.py

```diff
@@ -1385,9 +1385,11 @@ def __init__(self, cubelist, xunit='GHz', x0=0, y0=0, maskmap=None, **kwargs):
         self.data = self.cube[:,y0,x0]
         self.error = self.errorcube[:,y0,x0] if self.errorcube is not None else None
 
-        self.header = cubelist[0].header
+        self.header = cubelist[0].header.copy()
         for cube in cubelist:
             for key,value in cube.header.items():
+                if key in ['HISTORY', 'COMMENT']:
+                    continue
```

Comment:
Now that I think of it these two lines do remove duplicate values, but they also deny any history or comment entries from the rest of `cubelist`'s headers to be included. Should I get rid of this check?